src/pages/Routes.tsx

```diff
@@ -13,7 +13,12 @@ import { RedirectToSwap } from './Swap/redirects'
 const Rewards = lazy(() => import(/* webpackPrefetch: true */ './Rewards'))
 const Pools = lazy(() => import(/* webpackPrefetch: true */ './Pools'))
 const LiquidityMiningCampaign = lazy(() => import(/* webpackPrefetch: true */ './Pools/LiquidityMiningCampaign'))
-const Pair = lazy(() => import(/* webpackPrefetch: true */ './Pools/Pair'))
+const Pair = lazy(() =>
+  import(/* webpackPrefetch: true */ './Pools/Pair').then(({ Pair }) => ({
+    default: Pair,
+  }))
+)
+
```

Comment:
I suggest to make the main Pages as default export. Its like a weird way to do it. 

Even if we dont do default export other places. I feel we should do for our main routes. I am up to make all our Component exports default and all our hooks or utils as export.

Reply:
Thanks, I'll take a look into this.

Reply:
Fixed.